libs/features/webapp-prompt/focus-animation.css

```diff
@@ -0,0 +1,63 @@
+:root {
+  --blue-700: rgba(56,146,243);
+  --coach-indicator-ring-default-color:var(--blue-700);
+  --coach-indicator-ring-diameter:1.25rem;
+  --coach-indicator-ring-default-color:var(--blue-700);
+	--coach-indicator-ring-border-size: 2px;
+	--coach-indicator-ring-inline-size: var(--coach-indicator-ring-diameter);
+	--coach-indicator-ring-block-size: var(--coach-indicator-ring-diameter);
+  --animation-duration: 3000ms;
+}
+
+@keyframes pulse {
+	0% {
+		transform: scale(0.8);
+		opacity: 0;
+	}
+
+	50% {
+		transform: scale(1.5);
+		opacity: 1;
+	}
+
+	100% {
+		transform: scale(2);
+		opacity: 0;
+	}
+}
+
+.coach-indicator-ring {
+	display: block;
+	position: absolute;
+  top: 14%;
+  left: 13%;
+
+	border-style: solid;
+	border-width: var(--coach-indicator-ring-border-size);
+	border-color: var(--coach-indicator-ring-default-color);
+
+	inline-size: var(--coach-indicator-ring-inline-size);
+	block-size: var(--coach-indicator-ring-block-size);
+	animation: pulse var(--animation-duration) linear;
+  animation-fill-mode: both;
+
+  border-radius: 0.313rem;
```

Comment:
I'd argue that such values with so many decimals make it counter-intuitive to understand and update. Also, with exception of some quiz styles, all other border radius styles are defined as pixel values, it might be a good idea to stick to that pattern, as related upcoming Spectrum tokens will also be defined in pixels.

Reply:
You're right I've changed it to pixels. I've kept the `rem` units in `tooltip.css` that deal with text, though I converted the `border-radius` there to pixels as well.